ATTACK/Assets/Scripts/Projectiles/DamageScript.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DamageScript : MonoBehaviour
+{
+    private Team belongsTo;
+    public float timeToHit;
+    public float distance;
+    public int damage;
+
+    void FixedUpdate()
+    {
+        transform.Translate(Vector3.forward * Time.deltaTime * (distance/timeToHit));
+    }
+
+    public void SetTeam(Team team)
+    {
+        belongsTo = team;
+    }
+
+    void OnTriggerEnter(Collider other) {
+        CharacterCommon enemy = other.transform.GetComponent<CharacterCommon>();
+
+        if (enemy != null){
+            if (enemy.GetTeam() != belongsTo)
+            {
+                enemy.TakeDamage(damage);
+            }
+            else
+            {
+                Debug.Log("Hit Something else!");
```

Comment:
Maybe you could remove this Log? It will simply bloat our console if projectiles hit each other or something similar. What happens if one of those hidden projectile colliders hits an airplane, which also has a collider but does not have a CharacterCommon? Will it crash?

Reply:
Right, that one doesn't trigger anymore (intentionally) but should definitely be removed